small-multiple-column-timeline/parseData.js

```diff
@@ -0,0 +1,126 @@
+/**
+ * General data munging functionality
+ */
+
+import * as d3 from 'd3';
+
+/**
+ * Parses CSV file and returns structured data
+ * @param  {String} url Path to CSV file
+ * @return {Object}     Object containing series names, value extent and raw data object
+ */
+export function fromCSV(url, dateStructure, options) {
+    return new Promise((resolve, reject) => {
+        d3.csv(url, (error, data) => {
+            if (error) reject(error);
+            else {
+                const { yMin, joinPoints, highlightNames, dataDivisor } = options;
```

Comment:
Neither `highlightNames` or `joinPoints` is ever used. If these values are exposed for the sake of documentation, add a `// eslint-disable-line` comment at the end of this line.